styles.css

```diff
@@ -1,3 +1,131 @@
 * {
 	box-sizing: border-box;
+}
+body {
+	background-color: #CCCCFF;
+	background-image: url('img/ship-7679100.jpg');
+	background-size: cover;
+	height: 100vh;
```

Comment:
not sure if you need this? toggled off in inspector and stayed the same

Reply:
just double checked, but if I take this out the background image starts to repeat vertically